modules/proc_installing-the-rollout-manager-plugin.adoc

```diff
@@ -0,0 +1,106 @@
+:_mod-docs-content-type: PROCEDURE
+[id="installing-the-rollout-manager-plugin_{context}"]
+= Installing the rollout-manager plugin
+
+Once the rollout controller is created, you must install the rollout-manager plugin.
+
+.Prerequisites
+
+* The `cluster-admin` cluster role must be assigned to the user.
+
+.Procedure
+
+. Run the `yaml/rbac.yaml` file to add the role for running the required operations on the Openshift Route and build this plugin.
+
+. Save this plugin in your local system and mount it on the argo-rollouts container:
+
+.. Apply the rbac permissions:
++
+[source,terminal]
+----
+kubectl apply -f https://raw.githubusercontent.com/argoproj-labs/rollouts-plugin-trafficrouter-openshift/main/yaml/rbac.yaml
+----
+
+.. Create an instance of argo-rollouts by creating a file with the following configuration:
```

Comment:
As discussed during the call with Ishita, remove this step as it is not required now.